apps/atomy-q/API/tests/Unit/Services/VendorGovernanceScoreServiceTest.php

```
@@ -0,0 +1,78 @@
+            'compliance_health_score' => 100,
+            'risk_watch_score' => 100,
+            'evidence_freshness_score' => 100,
+        ], $summary['scores']);
+        $this->assertSame([], $summary['warning_flags']);
+    }
+
+    #[Test]
+    public function scoresExpiredEvidenceAndOpenSevereFindingsWithWarnings(): void
+    {
+        $service = new VendorGovernanceScoreService();
+        $now = CarbonImmutable::parse('2026-04-22T00:00:00Z');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Carbon\CarbonImmutable' in method 'scoresExpiredEvidenceAndOpenSevereFindingsWithWarnings'. (undefined)

(StaticAccess)